tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1970,3 +1970,155 @@ def _(
+        "trtllm::quantize_e4m3_per_tensor",
+        [quantize_runner],
+        QuantizeE4M3PerTensorRunner.tuning_config,
+        [input],
+    )
+
+    return quantize_runner(inputs=[input], tactic=best_tactic)
+
+
+@quantize_e4m3_per_tensor.register_fake
+def _(input: torch.Tensor,
+      tune_max_num_tokens: int = 8192) -> Tuple[torch.Tensor, torch.Tensor]:
```

Ruff (0.14.14)

Unused function argument: `tune_max_num_tokens`

(ARG001)